.github/workflows/public-api-check.yml

```diff
@@ -0,0 +1,31 @@
+name: Public API check
+
+on:
+  pull_request:
+
+jobs:
+  commentPR:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mshick/add-pr-comment@v1
+        with:
+          message: |
+            **comment**
```

Comment:
Are you still in the process of implementing this?

Reply:
I think I am done now :sweat_smile: